pages/linux/machinectl.md

```diff
@@ -0,0 +1,28 @@
+# Machinectl
```

Comment:
```suggestion
# machinectl
```

Note: We don't capitalize the words in page headings generally (as we keep it the same as case as the command)